### DISCUSSION TITLE
如何将vercel创建的库与主项目同步

### DISCUSSION THREAD

**Discussion by shoucandanghehe** [spencerwooo/onedrive-vercel-index#184] [Q&A] [answered]
vercel似乎并不会主动推送提交或者pr
只能手动同步吗？

**Comment by myl7** [comment, marked as answer]
就像 `apt update && apt upgrade` 一样，考虑到新的改动可能会 break 用户的应用，手动同步会更稳定一些
目前此项目还在 beta 阶段（还没 release 1.0），待之后项目成熟后，可以 watch GitHub release 来接受更新提醒
